src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/TransitTuningParameters.java

```diff
@@ -77,6 +82,8 @@ public List<RouteRequest> transferCacheRequests() {
    */
   int transferCacheMaxSize();
 
+  Duration maxSearchWindow();
```

Comment:
All `public` methods should have JavaDoc. What is the intention and why?

Reply:
Clarified both the Javadoc and the configuration documentation.